var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -681,6 +719,9 @@ def cmake_args(self):
                     "-DFFTWF_INCLUDE_DIR={0}".format(self.spec["acfl"].headers.directories[0])
                 )
                 options.append("-DFFTWF_LIBRARY={0}".format(self.spec["acfl"].libs.joined(";")))
+            elif self.pkg.version >= Version("2023") and "+rocm" in self.spec:
+                # Use ROCm FFT library
+                options.append("-DGMX_GPU_FFT_LIBRARY=rocFFT")
```

Comment:
Would be good to have it configurable (also `depends_on("rocfft")` above). At least by default, GROMACS suggests VkFFT when using ACpp/ROCm without HeFFTe, so it's a bit counter-intuitive that the Spack package has a different default.

Reply:
Good point. I added a variant `rocfft`.